src/inc/volatile.h

```diff
@@ -120,7 +120,7 @@ T VolatileLoad(T const * pt)
     STATIC_CONTRACT_SUPPORTS_DAC_HOST_ONLY;
 
 #ifndef DACCESS_COMPILE
-#if defined(_ARM64_) && defined(__GNUC__)
+#if defined(_ARM64_) && defined(__GNUC__) && !(defined(__BIONIC__))
```

Comment:
@cydhaselton Seems like `__atomic_store` later in the file should have the same issue.  For reference I cross compile this on Clang 3.8 targeting Ubuntu Xenial arm64.  It is not obvious why I did not see the issue.